_security/configuration/index.md

```diff
@@ -13,16 +13,96 @@ redirect_from:
 
 The plugin includes demo certificates so that you can get up and running quickly. To use OpenSearch in a production environment, you must configure it manually:
```

Comment:
```suggestion
The plugin includes demo certificates so that you can get up and running quickly. To use OpenSearch in a production environment, you must configure it manually.
```